owncloud/ios-app
# [fix/disable-move-copy] Disable move/copy action in toolbar if no folder is available

## Description
If there is no folder available in the root folder and a file is selected, the toolbar actions move/copy are disabled, because items cannot moved or copied to an other directory.

Furthermore this is also important for:
- Menu Cardview
- Drag and Drop on toolbar items

## Related Issue
#364

## Motivation and Context
Do not show user options, which are not possible.

## How Has This Been Tested?
- open file list
- be sure, there is no directory available in the root folder
- select a file
- toolbar actions move and copy should be disabled
- create a new folder
- select file again
- toolbar actions move and copy should be enabled

**Repeat same tests for Drag & Drop and Card Menu**

## Types of changes


## Checklist:
